shiny/api-examples/layout_columns/app.py

```diff
@@ -0,0 +1,33 @@
+from model_plots import *  # model plots and cards
+
+from shiny import App, Inputs, Outputs, Session, render, ui
+
+app_ui = ui.page_fluid(
+    ui.panel_title(ui.h2("Model Dashboard")),
+    ui.div("Using", ui.code("ui.layout_columns()"), ".", class_="mb-2"),
```

Comment:
Could use `ui.markdown()` here (and handle spacing on the `layout_columns()` instead?

Reply:
There's something up with the spacing on `layout_columns()` that I need to look into.